code/scripts/espurna_utils/bsec.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+def bsec_inject_patcher(env):
+    platform = env.PioPlatform()
+    framework_dir = platform.get_package_dir("framework-arduinoespressif8266")
+    bsec_dir = os.path.join(
```

Comment:
See `env.GetLibBuilders()`
`name` is what it gets from library.properties / library.json
`src_dir` is the path